docs/reference/commandline/dockerd.md

```diff
@@ -780,6 +785,113 @@ conditions the user can pass any size less then the backing fs size.
 $ sudo dockerd -s overlay2 --storage-opt overlay2.size=1G
 ```
 
+
+#### Windowsfilter options
+
+##### `size`
+
+Specifies the size to use when creating the sandbox which is used for containers.
+Defaults to 20G.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt size=40G
+```
+
+#### LCOW (Linux Containers on Windows) options
+
+##### `lcow.globalmode`
```

Comment:
Not sure I understand the question here..... Merging as per the PR which made this documentation followup? If so, yes. And on LCOW, `runtime.GOOS=="windows"`, yes. It's still a daemon running on Windows. Just the target containers (and images) are Linux ones.